core-web/libs/sdk/react/src/lib/components/Container/Container.tsx

```diff
@@ -1,26 +1,27 @@
 import { useContext } from 'react';
 
 import { PageContext } from '../../contexts/PageContext';
+import { DotCMSPageContext } from '../../models';
 import { getContainersData } from '../../utils/utils';
-import { PageProviderContext } from '../PageProvider/PageProvider';
 
 function NoContent({ contentType }: { readonly contentType: string }) {
     return <div data-testid="no-component">No Component for {contentType}</div>;
 }
 
 export interface ContainerProps {
-    readonly containerRef: PageProviderContext['layout']['body']['rows'][0]['columns'][0]['containers'][0];
+    readonly containerRef: DotCMSPageContext['pageAsset']['layout']['body']['rows'][0]['columns'][0]['containers'][0];
```

Comment:
Why dont we have a type for the PageAsset?

Reply:
We have all these types in @dotcms/angular

But until we have the type library, we cannot "duplicate" them or we will have SonarQ issues